### PR TITLE
Cordova Type Update - InAppBrowser.d.ts - Correct non-specialized overload for TypeScript 2.0

### PR DESCRIPTION
**Pull request by jicongw**
Cordova Type Update - InAppBrowser.d.ts - Correct non-specialized overload for TypeScript 2.0

### CODE REVIEW

File: cordova/plugins/InAppBrowser.d.ts [hunk 1]
```diff
@@ -152,9 +152,9 @@ interface InAppBrowser extends Window {
      *                  loaderror: event fires when the InAppBrowser encounters an error when loading a URL.
      *                  exit: event fires when the InAppBrowser window is closed.
      * @param callback  the function that executes when the event fires. The function is
-     *                  passed an InAppBrowserEvent object as a parameter.
+     *                  passed an Event object as a parameter.
      */
-    removeEventListener(type: "exit", callback: (event: InAppBrowserEvent) => void): void;
+    removeEventListener(type: "exit", callback: (event: Event) => void): void;
```
Comment (DanielRosenwasser) [inline cordova/plugins/InAppBrowser.d.ts]: Were these necessary? If you're entirely sure that `"exit"` events will give you back an `InAppBrowserEvent`, then those should stay the same.

The only ones you *can't* be sure about is the non-specialized signature (i.e. the one whose `type` parameter doesn't have a string literal type)`. So those should take `Event`.

**Comment by DanielRosenwasser** [comment]
@vvakame any idea what's going on with @dt-bot? The file doesn't appear to have any syntax errors, and checks are passing.

File: cordova/plugins/InAppBrowser.d.ts (hunk 1 above)
Comment (jicongw) [inline cordova/plugins/InAppBrowser.d.ts]: These are necessary. Without these change, the project won't compile.

Comment (DanielRosenwasser) [inline cordova/plugins/InAppBrowser.d.ts]: Can you elaborate? What errors are you getting?

**Comment by jicongw** [comment]
Creating a new PR after discussion with Daniel offline. Sending a new PR with updated fix.

**Pull request merged**